README.md

```diff
@@ -1,2 +1,190 @@
-# mmw
-Miami Music Weekend
+<a name="readme-top"></a>
+  
+
+<div align="center">
+  <br/>
+
+  <h1><b>MIAMI MUSIC WEEKEND</b></h1>
+
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Miami Music Weekend ](#-miami-music-weekend-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [⭐️ Key Features ](#️-key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Walkthrough Video](#walkthrough-video)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+  - [🛠 Built With ](#-built-with-)
+
+    - [Tech Stack ](#tech-stack-)
+
+  - [⭐️ Key Features ](#️-key-features-)
+
+  - [🚀 Live Demo ](#-live-demo-)
+
+  - [💻 Getting Started ](#-getting-started-)
+
+    - [Walkthrough Video](#walkthrough-video)
```

Comment:
- [ ] You did well keeping the video length less than or equal to 5mins as mentioned by the previous reviewer. But I did not see you mention and show/highlight a piece of code that you are proud of. Please make a video that also addresses that.

Reply:
**I recorded a new one and more professional walkthrough video.**

you can wacth it on the readme.md link or [here](https://youtu.be/ZjL0LeZlHdc)
-----
Thank you.